compose.yaml

```diff
@@ -0,0 +1,217 @@
+    entrypoint: /bin/sh
+    command: >
+      -c "
+      export PGPASSWORD=$$POSTGRES_PASSWORD;
+      echo \"SELECT 'CREATE DATABASE $$POSTGRES_DB' WHERE NOT EXISTS (SELECT FROM pg_database WHERE datname = '$$POSTGRES_DB')\\gexec\" |
+      psql -U $$POSTGRES_USER -h db
+      "
+    depends_on:
+      db:
+        condition: service_healthy
+
+  db:
+    image: postgis/postgis:17-3.5-alpine
+    environment:
+      - POSTGRES_USER=${POSTGRES_USER}
+      - POSTGRES_PASSWORD=${POSTGRES_PASSWORD}
+      - POSTGRES_DB=${POSTGRES_DB}
+    healthcheck:
+      test: pg_isready -U $$POSTGRES_USER -h localhost
+      interval: 5s
+      timeout: 5s
+      retries: 10
+    volumes:
+      - db_data:/bitnami/postgresql
+    ports:
+      - "127.0.0.1:5432:5432"
+
+  pgadmin:
+    image: dpage/pgadmin4:9.3.0
+    entrypoint: /bin/sh
+    command: >
+      -c "echo \"$$SERVERS_JSON\" > /tmp/servers.json &&
+      exec /entrypoint.sh"
+    environment:
+      SERVERS_JSON: |
+        {
+          "Servers": {
+            "1": {
+              "Name": "Mrija-Manager",
+              "Group": "Servers",
+              "Port": 5432,
+              "Username": "${POSTGRES_USER}",
+              "PasswordExecCommand": "echo '${POSTGRES_PASSWORD}'",
+              "Host": "db",
+              "SSLMode": "prefer",
+              "MaintenanceDB": "postgres"
+            }
+          }
+        }
+      PGADMIN_SERVER_JSON_FILE: /tmp/servers.json
+      PGADMIN_DEFAULT_EMAIL: 1@1.com
+      PGADMIN_DEFAULT_PASSWORD: "1"
+      PGADMIN_DISABLE_POSTFIX: "true"
+      PGADMIN_CONFIG_SERVER_MODE: "False"
+      PGADMIN_CONFIG_MASTER_PASSWORD_REQUIRED: "False"
+    depends_on:
+      db:
+        condition: service_healthy
+    labels:
+      - "traefik.enable=true"
+      - "traefik.http.routers.pgadmin.rule=Host(`pgadmin.localhost`)"
+      - "traefik.http.routers.pgadmin.entrypoints=web"
+
+  # ors-app:
+  #   image: openrouteservice/openrouteservice:v9.0.0
+  #   ports:
+  #     - "8080:8082"
+  #     - "9001:9001"
+  #   user: "1000:1000" # Default is 1000:1000. make infra/up will override this to the actual user values
+  #   volumes:
+  #     - ./.docker/infra/ors:/home/ors
+  #   environment:
+  #     ors.engine.profile_default.build.source_file: /home/ors/files/sh.osm.pbf
+  #     REBUILD_GRAPHS: false
+  #     CONTAINER_LOG_LAVEL: INFO
+  #     XMS: 256m
+  #     XMX: 512m
+  #     ADDITIONAL_JAVA_OPTS: ""
+  #   deploy:
+  #     resources:
+  #       reservations:
+  #         cpus: "1"
+  #         memory: 256m
+  #       limits:
+  #         cpus: "1"
+  #         memory: 512m
+  #   networks:
+  #     - app-network
+
+  vroom:
+    image: ghcr.io/vroom-project/vroom-docker:v1.14.0
+    volumes:
+      - ./.docker/infra/vroom:/conf
+    environment:
+      VROOM_ROUTER: valhalla
+    deploy:
+      resources:
+        reservations:
+          cpus: "1"
+          memory: 50m
+        limits:
+          cpus: "1"
+          memory: 50m
```

Comment:
With this much memory, the container crashed twice on my laptop right at the beginning. I think it can be loosened a bit.